AdminWebsite/AdminWebsite/ClientApp/tslint.json

```diff
@@ -72,7 +72,6 @@
     "no-trailing-whitespace": true,
     "no-unnecessary-initializer": true,
     "no-unused-expression": true,
-    "no-use-before-declare": false,
```

Comment:
this should stay, this style has been applied across the board, if any file isn't following this pattern the variable just needs to be re-ordered

Reply:
okay will try rerunning linting and turn this back on

Reply:
Done